run.go

```diff
@@ -1060,23 +1060,25 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	}
 	rootIDPair := &idtools.IDPair{UID: int(rootUID), GID: int(rootGID)}
 
-	hostFile, err := b.addNetworkConfig(path, "/etc/hosts", rootIDPair)
-	if err != nil {
-		return err
-	}
-	resolvFile, err := b.addNetworkConfig(path, "/etc/resolv.conf", rootIDPair)
-	if err != nil {
-		return err
-	}
+	bindFiles := make(map[string]string)
+	if configureNetwork {
```

Comment:
Is configureNetwork set for --net=host?  I thought it was turned off for that as well.  
I might be mistaken, but I saw little difference internally between 

--net=host
and
--net=none (This one actually did not work.  I was still able to do a yum update).

Reply:
Good point.  That controls whether or not we call CNI to do things, so it isn't.

Reply:
In rootless mode, we had been discarding `--net=none` by forcing `--net=host` for every case.  Was this how you checked?

Reply:
Added a change here to only discard the part of the configuration that specifies configuration names - whether or not we're using the host's network namespace, and if we're either leaving it unconfigured or attempting to attach to another namespace by specifying its path, should be preserved now for rootless containers.  @TomSweeneyRedHat is this what you were thinking?

Reply:
@nalind, yep, was basing my question off of what @rhatdan had done in https://github.com/containers/libpod/pull/1734.  I'd not seen Dan's comment here until after I punched mine in also.